iqm5q.py

```diff
@@ -3,14 +3,16 @@
 
 from qibolab import Platform
 from qibolab.channels import Channel, ChannelMap
-from qibolab.instruments.erasynth import ERA
 from qibolab.instruments.oscillator import LocalOscillator
+
+# from qibolab.instruments.erasynth import ERA
+from qibolab.instruments.rohde_schwarz import SGS100A
 from qibolab.instruments.zhinst import Zurich
 from qibolab.serialize import load_qubits, load_runcard, load_settings
 
 RUNCARD = pathlib.Path(__file__).parent / "iqm5q.yml"
 
-TWPA_ADDRESS = "192.168.0.210"
+TWPA_ADDRESS = "192.168.0.32"  # "192.168.0.210"
```

Comment:
```suggestion
TWPA_ADDRESS = "192.168.0.32" 
```

Let's remove the old IP

Reply:
done